### ISSUE TITLE
Potential improvements regarding the process to find app root

### ISSUE THREAD

**Issue by 0x7FFFFFFFFFFFFFFF** [portapps/portapps#122]
I noticed the following code has been used to find the app root path.

https://github.com/portapps/portapps/blob/7911c926c30bb2f767cbe84c19cac7fec2bed39a/portapps.go#L82-L86

When we double click the the generated exe (like idea.exe), this code always work as expected. However, when we add the portable app's root folder (like `C:\portapps\intellij-idea-ultimate-portable`) to the PATH environment variable and execute something like `idea my_app` or `idea .`, this piece of code will not work correctly. It will use the current directory as app root path. To solve this problem, we need to find the executable in PATH, then remove the file name and we get the correct app root path. I suggest we change the aforementioned snippet to this

```go
	// Find the executable in the PATH environment variable
	path, err := exec.LookPath(os.Args[0])
	if err != nil {
		app.FatalBox(errors.Wrap(err, "Cannot find executable in path"))
	}
	app.RootPath = filepath.Dir(path)
```

I've tested on my machine and it can correctly handle mouse-click launch as well as commands like `idea my_app` or `idea .`.

**Comment by crazy-max** [comment]
@0x7FFFFFFFFFFFFFFF LGTM, thanks for your feedback!

**Comment by crazy-max** [comment]
@0x7FFFFFFFFFFFFFFF I have found another way to get a proper path.  I will push a fix about this.

**Comment by crazy-max** [comment]
Fixed through 2.5.0
